packages/nx-playwright/src/executors/playwright-executor/lib/start-dev-server.ts

```diff
@@ -20,7 +20,7 @@ export const startDevServer = async (opts: PlaywrightExecutorSchema, context: Ex
   async function unwrapAsyncIterableIteratorPromise() {
     for await (const output of await runExecutor<ExecutorResult>(
       { project, target, configuration },
-      { hostname: 'localhost' },
+      {},
```

Comment:
`hostname` is not supported by all types of app executor and should be put into the `serve` section of app `project.json` if required